test/agent/agent_test.rb

```diff
@@ -1,7 +1,10 @@
 require 'test_helper'
-require 'oj'
+require './lib/oj_check'
 
 class AgentTest < Minitest::Test
+
+  Oj = ::Instana::Oj
```

Comment:
We may need a conditional here?  If `Oj` is available, `::Instana::Oj` isn't defined.   Travis tests show this too with: https://travis-ci.org/instana/ruby-sensor/jobs/572962547#L441